README.md

```diff
@@ -76,4 +83,78 @@ $ singularity build osgvo-pilot.sif docker://opensciencegrid/osgvo-docker-pilot
 ```
 
 
+## CVMFS Without a Bind-Mount Using cvmfsexec
+
+If you don't have CVMFS available on the host, the container can still make
+CVMFS available by using [cvmfsexec](https://github.com/cvmfs/cvmfsexec#readme).
+
+You will need to specify a list of CVMFS repos to mount in the environment
+variable `CVMFSEXEC_REPOS`.
+This will require a kernel version >= 3.10.0-1127 on an EL7-compatible host
+with user namespaces enabled or >= 4.18 on an EL8-compatible host --
+see the cvmfsexec README linked above for details.
+
+This will also require granting the container some additional privileges, which
+you can do in one of two ways:
+
+1.  Add `--privileged` to the `docker run` invocation.
+
+2.  Add
+    `--security-opt seccomp=unconfined --security-opt systempaths=unconfined --device=/dev/fuse`
+    to the `docker run` invocation.
```

Comment:
What's the use case for 1 vs 2?

Reply:
The first case is if system admins are comfortable with the risk and don't want to enable unprivileged user namespaces.  The second case is less risky but requires enabling unprivileged user namespaces.

Reply:
2 uses fewer privileges but it might not be possible to do in Kubernetes (seccomp requires the cluster admin to set something up, and I found absolutely nothing on systempaths).

Reply:
@matyasselmeci please elaborate on 1 vs 2 using your + Dave's comments